### ISSUE TITLE
bug on size on 1080x1920 screen

### ISSUE THREAD

**Issue by D3ENNY** [rototrash/tokyo-night-sddm#5]
I have an issue when i visualize the theme on `1080x1920` screen. 
The text size and some element size as the pointer are to big. 
[here a photo where it shows the issue:](https://imgur.com/EdvoiIH)

in 4k screen that issue dosn't occure:
[here a photo of 4k screen](https://imgur.com/1YqojSp)

ps: if imgur says to you that are img 18+ it's not true, pls trust me

here the theme.conf on [nekobin](https://nekobin.com/tupuyutaza)

**Comment by siddrs** [comment]
Hello,
Do you have all the required dependencies installed? Also looks like you don't have JetBrains Mono Nerd Font installed. The font is set as JetBrains Mono Nerd Font in theme.conf

Try Installing the said font (or change the font in theme.conf) and check if it solves the issue, else you can play around with the Font Size in theme.conf

**Comment by D3ENNY** [comment]
> Hello, Do you have all the required dependencies installed? Also looks like you don't have JetBrains Mono Nerd Font installed. The font is set as JetBrains Mono Nerd Font in theme.conf
> 
> Try Installing the said font (or change the font in theme.conf) and check if it solves the issue, else you can play around with the Font Size in theme.conf

yes, i have the dependencies installed, i have installed the `ttf-jetbrains-mono-nerd ` from repository `extra` of arch linux, but nothing is changed

**Comment by D3ENNY** [comment]
with settings font size to 7 the i resolve it, but the mouse pointer it remains bigger
